lib/portlayer/storage/vsphere/volume.go

```diff
@@ -191,3 +193,26 @@ func (v *VolumeStore) VolumesList(op trace.Operation) ([]*storage.Volume, error)
 
 	return volumes, nil
 }
+
+func (v *VolumeStore) Read(op trace.Operation, store *url.URL, child, ancestor string, pathspec map[string]string, data bool) (io.ReadCloser, error) {
+	_, err := util.VolumeStoreName(store)
+	if err != nil {
+		return nil, err
+	}
+
+	if ancestor == "" {
+		ancestor = storage.Scratch.ID
+	}
+
+	c := v.volDiskDsURL(child)
+	a := v.volDiskDsURL(ancestor)
```

Comment:
why are those called URL but image store ones called Path? Is there a difference?